core/kits/documents/tabs/theme-style-form-fields.php

```diff
@@ -136,16 +136,14 @@ protected function register_tab_controls() {
 		$this->add_control(
 			'form_field_focus_transition_duration',
 			[
-				'label' => esc_html__( 'Transition Duration', 'elementor' ) . ' (ms)',
+				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
-				'selectors' => [
-					$input_selector => 'transition: {{SIZE}}ms',
+				'size_units' => [ 's', 'ms' ],
+				'default' => [
+					'unit' => 'ms',
```

Comment:
@rami-elementor BC? What will happen for the old setting?

Reply:
The old selector had hard-coded unit `{{SIZE}}ms`. For BC, I replaced the selector with `{{SIZE}}{{UNIT}}` and added a default of `'unit' => 'ms'`.